samcli/lib/init/local_manifest.json

```diff
@@ -0,0 +1,176 @@
+{
+    "dotnetcore3.1": [
```

Comment:
Should we get rid of the runtimes no longer supported by lambda from this manifest? @mndeveci

Reply:
Agreed, we are working on to remove them from app template repo, let's don't include them here at all.